File: src/main/java/ksp/vilius/usersupport/Payload/UserRequest.java
package ksp.vilius.usersupport.Payload;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.web.multipart.MultipartFile;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserRequest {

    String firstname;
    String lastname;
    String username;
    String email;
    String role;
    boolean isNonLocked;
    boolean isActive;
    MultipartFile profileImage;

}

File: frontend/src/app/model/UserRequest.ts
export class UserRequest {
  firstName: string;
  lastName: string;
  username: string;
  email: string;
  role: string;
  notLocked: boolean;
  active: boolean;
  profileImage: File;

  constructor() {
    this.firstName = '';
    this.lastName = '';
    this.username = '';
    this.email = '';
    this.role = '';
    this.notLocked = false;
    this.active = false;
    this.profileImage = null;
  }
}
